Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: heal player cheat

### Description (*)

This PR adds a cheat to completely heal the player's character.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2209](https://jira.unity3d.com/browse/MTT-2209)
### Manual testing scenarios

1. Start a game and progress to the BossRoom scene
2. Let an imp damage your character
3. When you are damaged enough, open the cheat window and use the "Heal Player" button
4. See that you now have full hp
### Questions or comments

### Contribution checklist
